test/utils.ts

```diff
@@ -74,7 +73,7 @@ export class FileLoader implements Loader {
     const source = this.files[file] || "";
 
     if (fmTest(source, ["yaml"])) {
-      const { body, attrs } = extract(source);
+      const { body, attrs } = extract<Record<string, unknown>>(source);
```

Comment:
The added type annotation is necessary for the type checker to pass. This matches the previous behavior.

Perhaps we should add a check here to make sure that this actually is a `Record<string, unknown>` - technically one could do something like

```vto
---
23
---
Hello {{ name }}!
```

Probably fine either way, because I don't think people do this in practice, but could be nice to add a readable error message for this at some point.